tests/testthat/_snaps/cran-compliance.md

```diff
@@ -14,17 +14,15 @@
     Code
       package_versions
     Output
-      # A tibble: 10 x 2
-         crate          version
-         <chr>          <chr>  
-       1 build-print    *.*.*
-       2 extendr-api    *.*.*
-       3 extendr-ffi    *.*.*
-       4 extendr-macros *.*.*
-       5 once_cell      *.*.*
-       6 paste          *.*.*
-       7 proc-macro2    *.*.*
-       8 quote          *.*.*
-       9 syn            *.*.*
-      10 unicode-ident  *.*.*
+                  crate version
+      1     build-print   *.*.*
+      2     extendr-api   *.*.*
+      3     extendr-ffi   *.*.*
+      4  extendr-macros   *.*.*
+      5       once_cell  *.*.*
+      6           paste  *.*.*
+      7     proc-macro2  *.*.*
+      8           quote  *.*.*
+      9             syn *.*.*
+      10  unicode-ident  *.*.*
```

Comment:
Given that we don't actually test the versions here. I'm of the mind we should extract the crate column and check against a known vector. This snapshot in particular has been a bit...finicky. Though I suppose it is out of scope!

Reply:
I haven't noticed any flakyness, but I might be wrong. If it will constantly cause issues, yup, let's replace it